lnbits/wallets/__init__.py

```diff
@@ -16,6 +16,7 @@
 from .corelightningrest import CoreLightningRestWallet
 from .eclair import EclairWallet
 from .fake import FakeWallet
+from .getalby import AlbyWallet
```

Comment:
```suggestion
from .alby import AlbyWallet
```

+ rename `lnbits/wallets/getalby.py` to `lnbits/wallets/alby.py`

Reply:
OK, will rename. 

One thing that is not clear why its not populating is the Form Field, for Alby Wallet it does not populate. Am not sure what I am missing? Is there something I'm supposed to set in the javascript side?